frontend/web/components/DeleteGithubIntegration.tsx

```diff
@@ -37,7 +38,7 @@ const DeleteGithubIntegracion: FC<DeleteGithubIntegracionType> = ({
               github_integration_id: githubId,
               organisation_id: organisationId,
             }).then(() => {
-              onConfirm()
+              window.location.reload()
```

Comment:
Im confused, does this work ? It doesnt look like the correct way to open a confirmation modal, there's no message onYes etc

Reply:
This closes the modal when you delete the installation.